views/login.hbs

```diff
@@ -1,108 +1,72 @@
-				<div class="panel-heading">
-					<section class="col-xs-6">
-						<a href="#" class="active" id="login-form-link">Login</a>
-					</section>
-					<section class="col-xs-6">
-						<a href="#" id="register-form-link">Register</a>
-					</section>
-					<hr>
-				</div>
-				<div class="panel-body">
-					<div class="row">
-						<div class="col-lg-12">
-              {{!-- login form --}}
-							<form id="login-form" action="/api/loginUser" method="post" role="form" style="display: block;">
-								<section class="form-group">
-									<input type="text" name="username" id="usernamelogin" tabindex="1" class="form-control" placeholder="Username" value="">
-								</section>
-								<section class="form-group">
-									<input type="password" name="password" id="passwordlogin" tabindex="2" class="form-control" placeholder="Password">
-								</section>
-								<section class="form-group text-center">
-										<input type="checkbox" tabindex="3" class="" name="remember" id="remember">
-										<label for="remember"> Remember Me</label>
-								</section>
-									<div class="form-group">
-										<div class="row">
-											<div class="col-sm-6 col-sm-offset-3">
-												<input type="submit" name="login-submit" id="login-submit" tabindex="4" class="form-control btn btn-login" value="Log In">
-											</div>
-										</div>
-									</div>
-									<div class="form-group">
-										<div class="row">
-											<div class="col-lg-12">
-												<div class="text-center">
-													<a href="#" tabindex="5" class="forgot-password">Forgot Password?</a>
-												</div>
-											</div>
-										</div>
-									</div>
-								</form>
-                {{!-- register form --}}
-								<form id="register-form" action="api/newUser" method="post" role="form" style="display: none;">
-									<div class="form-group">
-										<input type="text" name="username" id="username" tabindex="1" class="form-control" placeholder="Username" value="">
-									</div>
-									<div class="form-group">
-										<input type="email" name="email" id="email" tabindex="1" class="form-control" placeholder="Email Address" value="">
-									</div>
-									<div class="form-group">
-										<input type="password" name="password" id="password" tabindex="2" class="form-control" placeholder="Password">
-									</div>
-									<div class="form-group">
-										<input type="password"id="confirm-password" tabindex="2" class="form-control" placeholder="Confirm Password">
-									</div>
-									<div class="form-group">
-										<div class="row">
-											<div class="col-sm-6 col-sm-offset-3">
-												<button type="submit" id="register-submit" tabindex="4"  class="form-control btn btn-register">Register Now</button>
-											</div>
-										</div>
-									</div>
-								</form>
-							</div>
+<!--add jquery-->
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```

Comment:
You can move script link to "layout.hbs".   login.hbs should contain only <body> part.

Reply:
in layout.hbs I saw that jquery code but because of adding bottom of body it doesn't work if put it on head it will work OK I remove it

Reply:
thank you so much